fastlane/lib/fastlane/actions/appetize.rb

```diff
@@ -22,6 +22,10 @@ def self.run(options)
             platform: 'ios'
         }
 
+        if options[:platform]
```

Comment:
When setting the default value for `platform` according to https://github.com/fastlane/fastlane/pull/4487/files#r61944033, you can remove this `if` statement